composer.json

```diff
@@ -16,19 +16,19 @@
     ],
     "require": {
         "php": "^7.3|^7.4|^8.0",
-        "illuminate/support": "~6.0|~7.0|~8.0",
```

Comment:
Do we want to maintain support for laravel 6/7?  Does this require a semver major bump, or only minor? (On my phone and it won't show me changes in a few of the files)

Reply:
Given the scope of the changes, I'd say a minor bump is probably prudent either way, so deprecating seems reasonable.  They're no longer supported by paragel anyway